analytics-api-core/src/main/scala/org/ekstep/analytics/api/util/auth_verifier/AccessTokenValidator.scala

```diff
@@ -0,0 +1,46 @@
+package org.ekstep.analytics.api.util.auth_verifier
+
+import java.nio.charset.StandardCharsets
+import org.ekstep.analytics.api.util.JSONUtils
+import java.util.Base64
+
+import javax.inject.Singleton
+import org.ekstep.analytics.framework.conf.AppConf
+
+@Singleton
+class AccessTokenValidator {
+
+    implicit val className = "org.ekstep.analytics.api.util.auth_verifier.AccessTokenValidator"
```

Comment:
Fix the package name. remove "auth_verifier' from the package name

Reply:
Done removed package name